### ISSUE TITLE
Cannot add box to vagrant

### ISSUE THREAD

**Issue by lukaisailovic** [laravel/homestead#605]
To me looks like this box is inaccessible for some reason. Whey I try command `vagrant box add laravel/homestead ` I get this error:
```
The box 'laravel/homestead' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Vagrant Cloud, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://vagrantcloud.com/laravel/homestead"]
Error: The requested URL returned error: 401 Unauthorized
```

Then I tried to do vagrant login and did log in successfully, but same thing

**Comment by svpernova09** [comment]
Must be a hiccup, it's working fine for me:

```
C:\Users\halo
λ vagrant box add laravel/homestead
==> box: Loading metadata for box 'laravel/homestead'
    box: URL: https://atlas.hashicorp.com/laravel/homestead
This box can work with multiple providers! The providers that it
can work with are listed below. Please review the list and choose
the provider you will be working with.

1) parallels
2) virtualbox
3) vmware_desktop

Enter your choice:```

**Comment by lukaisailovic** [comment]
It worked fine earlier in the day, but I had to reinstall my OS and for some reason after that its not working anymore...

**Comment by svpernova09** [comment]
This is definitely something on your end or else we'd be flooded with others having the same issue. Make sure you're running the most up to date version of vagrant.

**Comment by svpernova09** [comment]
If this is still an issue please let us know here.

**Comment by jimmerioles** [comment]
Reinstalling vagrant solved my issue.

**Comment by w-villena13** [comment]
i am having the same issue when try to run vagrant box add laravel/homestead

**Comment by browner12** [comment]
can you post the output of `vagrant -v`. Everything I've come across says this is a problem with an older version of Vagrant.

**Comment by chukkynze** [comment]
I had the same problem. I ran vagrant up and it could not find the homestead box:
`The box 'laravel/homestead' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/laravel/homestead"]
Error: The requested URL returned error: 404 Not Found
`

I solved it by updating my ubuntu 16.04 box and upgrading vagrant to version 2 and updating virtualbox to virtualbox-5.2. Vagrant downloaded the box successfully but not from the location stated above. I would infer that the location of the box was changed for the new version and the solution is just to upgrade everything
